presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -141,7 +141,7 @@ class SystemConfig : public ConfigBase {
   static constexpr int32_t kHttpExecThreadsDefault = 8;
   static constexpr bool kHttpServerHttpsEnabledDefault = false;
   static constexpr std::string_view kHttpsSupportedCiphersDefault{
-      "AES128-SHA,AES128-SHA256,AES256-GCM-SHA384"};
+      "ECDHE-ECDSA-AES256-GCM-SHA384,AES256-GCM-SHA384"};
```

Comment:
Changing default values to match deployment configuration. They can be passed as config parameter to overwrite during deployment as well.